src/ptvsd/_vendored/pydevd/pydevd.py

```diff
@@ -2253,6 +2253,7 @@ def _enable_attach(
     patch_multiprocessing=False,
     access_token=None,
     ide_access_token=None,
+    as_client=False
```

Comment:
I know this is already merged, but I don't think we should do this change on `pydevd`... the API to connect as a client is calling `settrace` directly with the proper host/port, whereas `_enable_attach()` was really only meant to be called to act as a server and wait until a connection was done, so, I think the changes on `pydevd` should be reverted and `ptvsd` should just use `settrace` directly here (the code related to the checking the `port` is also not interesting in this case... if it's starting a server it should be checked, but if it'll connect to a port in the client it could connect one time to one port and later on connect to another port).